client/src/app/site/base/base-sort.service/base-sort-list.service.ts

```diff
@@ -177,6 +179,8 @@ export abstract class BaseSortListService<V extends BaseViewModel>
 
     private initializationCount = 0;
 
+    private activeMeetingIdService = inject(ActiveMeetingIdService);
```

Comment:
Shouldn't this be injected via the constructor like all other services?

Reply:
I had problems with the use of constructor injection here, as I need to update  5+ super calls then. I talked to @bastianjoel  about this and he said I can use this method here.

Reply:
`inject` was introduced in Angular 14. I assume that this was not done before because the feature was not available when these classes were written. 
In my opinion, placing service injections in the constructor within base classes (or any other classes that will be extended) makes these classes unnecessarily harder to maintain as you always need to update all references when adding or removing a service the class is using.

Reply:
I agree! I just asked for consistency reasons. I'm all for switching all classes to the new injector style, if possible - the long constructor chains have always bugged me immensly. Is there maybe an automatic migration tool to get rid of these?